### PR TITLE
tests: TF AKS acceptance tests

### PR DESCRIPTION
**Pull request by aldor007**

### CODE REVIEW

File: castai/resource_node_template_test.go [hunk 1]
```diff
@@ -264,7 +266,7 @@ func testAccNodeTemplateConfig(rName, clusterName string) string {
 }
 
 func testNodeTemplateUpdated(rName, clusterName string) string {
-	return ConfigCompose(testAccClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
+	return ConfigCompose(testAccEKSClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
```
Comment (zilvinasu) [inline castai/resource_node_template_test.go:269]: EKS?

File: castai/resource_node_template_test.go [hunk 2]
```diff
@@ -230,7 +232,7 @@ func TestAccResourceNodeTemplate_basic(t *testing.T) {
 }
 
 func testAccNodeTemplateConfig(rName, clusterName string) string {
-	return ConfigCompose(testAccClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
+	return ConfigCompose(testAccEKSClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
```
Comment (zilvinasu) [inline castai/resource_node_template_test.go:235]: EKS -> AKS?

File: castai/resource_aks_cluster_test.go [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  "name": "aks-cluster",
+  "organizationId": "2836f775-aaaa-eeee-bbbb-3d3c29512692",
+  "credentialsId": "9b8d0456-177b-4a3d-b162-e68030d656aa",
+  "createdAt": "2022-01-27T19:03:31.570829Z",
+  "status": "ready",
+  "agentSnapshotReceivedAt": "2022-03-21T10:33:56.192020Z",
+  "agentStatus": "online",
+  "providerType": "eks",
+  "aks": {
+	"maxPodsPerNode": 100,
```
Comment (zilvinasu) [inline castai/resource_aks_cluster_test.go:47]: indent for maxPodsPerNode prop?

File: castai/resource_node_template_test.go (hunk 1 above)
Comment (aldor007) [reply castai/resource_node_template_test.go:269]: The name is correct. I've created a new `testAccClusterConfig` method, to not bring confusion I've added the prefix `AKS`. So now we have `testAccEKSClusterConfig` and `testAccAKSClusterConfig`

**Review by zilvinasu — APPROVED** [review]

File: castai/resource_node_template_test.go (hunk 1 above)
Comment (zilvinasu) [reply castai/resource_node_template_test.go:269]: got it, clear

**Review by zilvinasu — APPROVED** [review]

**Review by varnastadeus — APPROVED** [review]

**Pull request merged**